stock-tracker/src/features/chart/redux/chartReducer.test.ts

```diff
@@ -0,0 +1,74 @@
+import { chartReducer, ChartState } from "./chartReducer";
+import { SET_CHART_DATA, SET_CHART_TIME, SET_ERROR_CHART } from "./actionTypes";
+import { ADD_SYMBOL } from "features/search/redux/actionTypes";
+import { ChartActions } from "./actions";
+import { SetSymbol } from "features/search/redux/actions";
+
+describe("testing chart reducer", () => {
+  let initialState: ChartState = {
+    chartData: [],
+    chartTime: "1Y",
+    loading: false,
+    error: false
+  };
+  let newState: ChartState;
+
+  describe("returns expected payload when SET_CHART_DATA action is called", () => {
+    const action: ChartActions = {
+      type: SET_CHART_DATA,
+      payload: [{ close: 1, date: "2019/11/22" }]
+    };
+
+    beforeAll(() => {
+      newState = chartReducer(initialState, action);
+    });
+
+    it("should update the state", () => {
+      expect(newState.chartData).toEqual([{ close: 1, date: "2019/11/22" }]);
+    });
+  });
+
+  describe("returns expected payload when SET_CHART_TIME action is called", () => {
```

Comment:
This should be more descriptive, so you could say `returns "5y`......

Reply:
I'd say this description as it is, is good - It describes what it's actually testing without going into the specifics. Internally it tests for "5y", but it could do the same test with any other string and it would still be testing the same.

Please Alex, try to use a softer tone in code reviews, avoid "must" or "should" when you're sharing your opinions.

Reply:
Apologies, this was just meant to be a question/suggestion, I'll be sure to reword comments on PRs in the future